### ISSUE TITLE
Crudini Win32x

### ISSUE THREAD

**Issue by pedroalbanese** [pixelb/crudini#68]
Hello; Would it be possible to explain step by step how to compile or Crudini in Cygwin under Win32bits? I tried but was unsuccessful.

Thanks.

**Comment by pixelb** [comment]
crudini is python, so you should just need python (and iniparse python module) available on your system

**Comment by thorstenkampe** [comment]
Crudini does work on Cygwin but not on plain Windows. Looks like a `setuptools` issue to me (no `exe` or `bat` wrapper created). See #35

**Comment by dragoangel** [comment]
Well it not works. Tried run under powershell:
`& python (Get-Command -Name crudini).Path --get $MyIniFIle section variable`
and in result:
```
Traceback (most recent call last):
  File "C:\ProgramData\Python\Python39\Scripts\crudini", line 979, in <module>
    sys.exit(main())
  File "C:\ProgramData\Python\Python39\Scripts\crudini", line 975, in main
    return crudini.run()
  File "C:\ProgramData\Python\Python39\Scripts\crudini", line 868, in run
    self.conf = self.parse_file(self.cfgfile,
  File "C:\ProgramData\Python\Python39\Scripts\crudini", line 647, in parse_file
    self.locked_file = LockedFile(filename, self.mode, self.inplace,
  File "C:\ProgramData\Python\Python39\Scripts\crudini", line 165, in __init__
    self.lock()
  File "C:\ProgramData\Python\Python39\Scripts\crudini", line 96, in lock
    msvcrt.locking(self.fp, msvcrt.LK_LOCK, 1)
TypeError: an integer is required (got type _io.TextIOWrapper)
```
Versions:
```
Python 3.9.0
crudini in c:\programdata\python\python39\lib\site-packages (0.9.3)
iniparse in c:\programdata\python\python39\lib\site-packages (0.5)

```

**Comment by pixelb** [comment]
Addressed in commit 0f2c0b03
